main.css

```diff
@@ -0,0 +1,68 @@
+html{
+  background-color: #BEBEBE
+}
+body {
+	background-color: #FFFFFF;
+	color: #333333;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  margin: auto;
+  width: 50%;
+  border-style: solid;
+  border-width: 1px;
+  border-color: #333333;
+  padding: 5px;
+}
+h1{
+  background-color: #F9F264;
+  font-family: "Trebuchet MS", Helvetica, sans-serif;
+  font-size: 20px;
+  padding-top: 30px;
+  padding-bottom: 30px;
+  padding-left: 5px;
+}
+img{
+  border: 2px solid #F9F264;
+  width: 40%;
+}
+.subheader{
+  font-size: 12px
+}
+h2{
+  border-bottom: 1.5px solid black;
+  text-transform: uppercase;
+  font-size: 16px;
+  font-family: "Trebuchet MS", Helvetica, sans-serif;
+  width: 80%;
+}
+ul.list {
+  list-style-type: none;
+  margin: 0;
+  padding: 5px;
+  line-height: 200%;
+}
+ul.list > li:nth-of-type(odd) {
+  border-bottom-style: dashed;
+  border-bottom-width: 1px;
+  width: 50%
+}
+ul.list > li:nth-of-type(even) {
+  border-bottom-style: dashed;
+  border-bottom-width: 1px;
+  width: 50%
+}
+li.list{
+  font-size: 30px;
+}
+ol.list {
+  line-height: 200%;
+  padding-left: 30px;
+}
+footer{
+  background-color: #333333;
+  font-size: 10px;
+  color: white;
+  padding-top: 30px;
+  padding-bottom: 30px;
+  padding-left: 30px;
+}
```

Comment:
What could we do to make the hyperlink color in your footer white instead of blue? Take a look at this reference for help if you need it! https://www.w3schools.com/css/css_link.asp